src/sentry/integrations/utils/code_mapping.py

```diff
@@ -399,13 +397,21 @@ def convert_stacktrace_frame_path_to_source_path(
     )
 
     if stacktrace_path and stacktrace_path.startswith(code_mapping.stack_root):
-        return stacktrace_path.replace(stack_root, code_mapping.source_root, 1)
+        return (
+            stacktrace_path.replace(stack_root, code_mapping.source_root, 1)
+            .lstrip("/")
+            .replace("\\", "/")
+        )
 
     # Some platforms only provide the file's name without folder paths, so we
     # need to use the absolute path instead. If the code mapping has a non-empty
     # stack_root value and it matches the absolute path, we do the mapping on it.
     if frame.abs_path and frame.abs_path.startswith(code_mapping.stack_root):
-        return frame.abs_path.replace(stack_root, code_mapping.source_root, 1)
+        return (
+            frame.abs_path.replace(stack_root, code_mapping.source_root, 1)
+            .lstrip("/")
```

Comment:
Might make more sense to `.lstrip("/")` after the replacement in case there is a leading backslash